x-pack/test_serverless/api_integration/test_suites/search/config.ts

```diff
@@ -14,4 +14,8 @@ export default createTestConfig({
     reportName: 'Serverless Search API Integration Tests',
   },
   suiteTags: { exclude: ['skipSvlSearch'] },
+
+  // include settings from project controller
+  // https://github.com/elastic/project-controller/tree/main/internal/project/esproject/config
```

Comment:
Probably the link should be
https://github.com/elastic/project-controller/blob/main/internal/project/esproject/config/elasticsearch.yml ?

Reply:
Good catch! Updated all URLs for the search project controller config in 35aa56e